cmd/controller/main.go

```diff
@@ -60,10 +60,6 @@ func init() {
 }
 
 func main() {
```

Comment:
Can you add a reference to where the other source of validation is?

Reply:
Options are validated when they are parsed. https://github.com/aws/karpenter/blob/75afd6006eeb60669995419d82a412e1e174ccd4/pkg/utils/options/options.go#L37-L39

Reply:
@ellistarn Shall we resolve it, or add some comments to the code?